cluster/juju/layers/kubernetes-master/config.yaml

```diff
@@ -30,3 +30,7 @@ options:
     type: string
     default: "RBAC"
     description: kube-apiserver --authorization-mode <value>
+  cluster-context:
+    type: string
+    default: "juju-context"
```

Comment:
Actually @ktsakalozos you can get the model name from the hookenv.

https://jujucharms.com/docs/stable/reference-environment-variables#juju_model_name